modules/demote.js

```diff
@@ -0,0 +1,90 @@
+const { MessageType } = require("@adiwajshing/baileys")
+const chalk = require('chalk');
+module.exports = {
+    name: "demote",
+    description: "Demote",
+    extendedDescription: "Demote member from admin",
+    async handle(client, chat , BotsApp, args){
+        if(!BotsApp.isGroup){
+            client.sendMessage(BotsApp.from, "*This is not a group*", MessageType.text);
+        }
+        else{
+            if(!BotsApp.isBotGroupAdmin){
+                client.sendMessage(BotsApp.from,"*I am not group admin*", MessageType.text);
+            }
+
+            else{
+                try{
+                    if(chat.messages.all()[0].message.extendedTextMessage.text.split("@")[1]== null){
+                        
+                        var adm = false;
+                        let parName = " "; 
+                        for (const key in BotsApp.groupMembers) {
+                            if(chat.messages.all()[0].message.extendedTextMessage.contextInfo.participant.split('@')[0] == BotsApp.groupMembers[key].id.split('@')[0]){
+                                parName = BotsApp.groupMembers[key].name;
+                                if(BotsApp.groupMembers[key].isAdmin){
+                                    adm = true;
+                            
+                                }
```

Comment:
We could instead do this, less expensive operation -
```suggestion
                                if(BotsApp.groupMembers[key].isAdmin){
                                       parName = BotsApp.groupMembers[key].name;
                                    adm = true;
                            
                                }
```